src/controllers/user.js

```diff
@@ -138,9 +138,14 @@ export default {
     return response.status(200).json({
       status: 'success',
       message: 'Request successful',
-      data,
-      count,
-      page
+      followers: data.map((item) => {
```

Comment:
followers should be data ie.e 
```suggestion
      data: whatever.map((item) => {
```

Reply:
Done.